prow/cmd/deck/main_test.go

```diff
@@ -34,10 +34,13 @@ import (
 
 	"github.com/ghodss/yaml"
 
+	"github.com/google/go-cmp/cmp"
```

Comment:
Instead of porting a net-new package just for doing semantic deep equal checks, you could bump k8s.io/apimachinery to include [pkg/api/equality](https://github.com/kubernetes/apimachinery/tree/18a564baac720819100827c16fdebcadb05b2d0d/pkg/api/equality) and use equality.Semantic.DeepEqual (which is going to be useful for doing deep equality checks with k8s objects elsewhere in our codebase).